plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/StandardSQLDialectQueryGenerator.java

```diff
@@ -38,6 +41,7 @@
 
 public class StandardSQLDialectQueryGenerator implements SQLDialectQueryGenerator {
     private Log log = Log.getLog(StandardSQLDialectQueryGenerator.class);
+    private static final String NESTED_QUERY_AlIAS = "z_q";
```

Comment:
What if you have nested query inside nested query?

Reply:
Same as old logic, I just transferred it into the new class to be able to overload it.